testutil/vault.go

```diff
@@ -36,6 +36,92 @@ type TestVault struct {
 
 // NewTestVault returns a new TestVault instance that has yet to be started
 func NewTestVault(t *testing.T) *TestVault {
+	for i := 10; i >= 0; i-- {
+		port := getPort()
+		token := structs.GenerateUUID()
+		bind := fmt.Sprintf("-dev-listen-address=127.0.0.1:%d", port)
+		http := fmt.Sprintf("http://127.0.0.1:%d", port)
+		root := fmt.Sprintf("-dev-root-token-id=%s", token)
+
+		bin := "vault"
+		if runtime.GOOS == "windows" {
+			bin = "vault.exe"
```

Comment:
Looks like this isn't necessary as exec.Command calls exec.LookPath which on Windows looks for files with the appropriate extension: https://golang.org/src/os/exec/lp_windows.go#L53